pubspec.yaml

```
@@ -1,6 +1,6 @@
 name: app_review
 description: Request and Write Reviews and Open Store Listing for Android and iOS in Flutter.
-version: 2.1.2+1
+version: 2.1.4
```

```suggestion
version: 2.1.3
```
This should match the changelog